### PR TITLE
Undefined variable $order_id

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**

### CODE REVIEW

**Review by sandesh556 — CHANGES_REQUESTED** [review]
why is this moved, it makes no sense. if you are getting no order_id just use optional

**Review by sandesh556 — APPROVED** [review]

File: app/Http/Controllers/Order/RenewController.php [hunk 1]
```diff
@@ -259,10 +259,10 @@ public function renew($id, Request $request)
             $payment_method = $request->input('payment_method');
             $code = $request->input('code');
             $cost = $request->input('cost');
+            $sub = Subscription::find($id);
```
Comment (RafficMohammed) [inline app/Http/Controllers/Order/RenewController.php:262]: $order_id = Subscription::whereid($id)->value('order_id');

**Review by RafficMohammed — CHANGES_REQUESTED** [review]

**Review by RafficMohammed — APPROVED** [review]

**Comment by Sabaric12** [comment]
Round 1: Testing Summary
Tested this PR with the server instance
The mentioned issue is working fine now if we click on renew fill in the mandatory fields and save after that the success message comes
Before Fix:
image
After Fix:
Screenshot from 2023-12-14 11-48-46
Total time in days: 2 hours
Total issues:0 
Minor - 0 
Major - 0 
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality:No

**Review by Sabaric12 — APPROVED** [review]

**Review by ganthimathin — APPROVED** [review]
Tested this issue with existing DB. Issue was fixed.

**Comment by ganthimathin** [comment]
Second round test summary:

Total time in days: 1 hour
Total issues:0
Minor - 0
Major - 0
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality:No

**Pull request merged**